notNeededPackages.json

```diff
@@ -7471,6 +7471,10 @@
             "libraryName": "tap",
             "asOfVersion": "18.0.0"
         },
+        "tapable": {
+            "libraryName": "tapable",
+            "asOfVersion": "2.2.8"
```

Comment:
This one somewhat concerns me, because tapable has types in v2 so if any lingering packages out there depend on `@types/tapable@v2`, now will require `tapable` and might stop working. But maybe it's fine given there's no v3?